react/features/welcome/components/WelcomePageSideBar.native.js

```diff
@@ -43,14 +42,14 @@ type Props = {
     dispatch: Function,
 
     /**
-     * The avatar URL to be rendered.
+     * Display name of the local participant.
      */
-    _avatarURL: string,
+    _displayName: string,
 
     /**
-     * Display name of the local participant.
+     * The local participant.
      */
-    _displayName: string,
+    _localParticipant: Object,
```

Comment:
Why are we passing the entire participant instead of just the ID?

Reply:
Dunno, because it was available. But you are right, I change it, thanks!